platform/AppStoreConnect/Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "AppStoreConnect",
+    platforms: [.macOS(.v10_15)],
+    products: [
+        .executable(name: "AppStoreConnectCMD", targets: ["AppStoreConnectCMD"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/AvdLee/appstoreconnect-swift-sdk.git", from: "1.7.0"),
+        .package(url: "https://github.com/apple/swift-argument-parser", from: "1.1.1")
```

Comment:
If we keep these dependencies we should create forks of them and update it here - directly depending on random people's GitHub repos is way too exciting :D

Reply:
well, at least the second dependency is an official one from apple 😄

Reply:
These should be probably OK...

Reply:
I am for forking the repos, but I don't think I have the privileges to do so.

Reply:
Well, I didn't catch that one of those is official from Apple - I **do** trust them to not do anything malicious. And I do trust GitHub to (probably) keep it up. So I'd skip over forking that one. But I don't trust _AvdLee_ since I don't know who that is. He could be pulling a left pad on us any day...

Forked it: https://github.com/polypoly-eu/appstoreconnect-swift-sdk